test/stdlib/DispatchData.swift

```diff
@@ -23,7 +25,13 @@ DispatchAPI.test("dispatch_data_t deallocator") {
 	let q = DispatchQueue(label: "dealloc queue")
 	var t = 0
 
-	autoreleasepool {
+#if os(macOS) || os(iOS)
+	let runWithPool: (()->())->() = { autoreleasepool { $0() } }
+#else
+	let runWithPool: (()->())->() = { $0() }
+#endif
```

Comment:
OMG, this is beautiful!  I ended up doing some more ... intensive refactoring in swift-syntax to deal with the autoreleasepool.